target/linux/ramips/image/mt7620.mk

```diff
@@ -804,6 +804,24 @@ define Device/netgear_wn3000rp-v3
 endef
 TARGET_DEVICES += netgear_wn3000rp-v3
 
+define Device/netgear_wn3100rp-v2
+  SOC := mt7620a
+  IMAGE_SIZE := 7872k
+  NETGEAR_HW_ID := 29764883+8+0+32+2x2+0
+  NETGEAR_BOARD_ID := WN3100RPv2
+  BLOCKSIZE := 4k
+  IMAGES += factory.bin
+  KERNEL := $(KERNEL_DTB) | uImage lzma | pad-offset 64k 64 | \
+	append-uImage-fakehdr filesystem
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size | netgear-dni
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := WN3100RP
+  DEVICE_VARIANT := v2
+  DEVICE_DTS := mt7620a_netgear_wn3000rp-v3
+  SUPPORTED_DEVICES += wn3100rpv2 wn3000rpv3
```

Comment:
Please remove this line. This line is not needed for newly added devices, and the images are not compatible anyway.

Reply:
I added it because I did encounter encounter a problem with sysupgrade.

Trying to upgrade the wn3100rpv2 image ended up in an error because the device now thinks it's a wn3000rpv3
Do you known where this information is computed/stored so the sysupgrade can work ?

Reply:
Hmm, I guess that problem will disappear once you reinstall a (proper) new image.

Reply:
Ok, I'll come back once I've applied the requested changes